### PR TITLE
documentation fixes

### PR DESCRIPTION
**Pull request by dtrawins**
http://openvino-doc.iotg.sclab.intel.com/dtrawins-test1/ovms_what_is_openvino_model_server.html

### CODE REVIEW

File: docs/advanced_topics.md [hunk 1]
```diff
@@ -25,4 +25,9 @@ Leverage the OpenVINO [model caching](https://docs.openvino.ai/latest/openvino_d
...
 Write your own custom model loader based on a predefined interface and load it similar to a dynamic library.  
 
-[Learn more](custom_model_loader.md)
\ No newline at end of file
+[Learn more](custom_model_loader.md)
+
+## Securing Model Server with NGINX
+Protect network endpoints with traffic encription and client authorization using a reverse proxy
+
+[Learn more](../extras/nginx-mtls-auth/README.md)
```
Comment (mzegla) [inline docs/advanced_topics.md:33]: Will this work in sphinx? Don't we need to link github url here?

File: docs/accelerators.md [hunk 2]
```diff
@@ -187,12 +191,15 @@ Make sure you have passed the devices and access to the devices you want to use
...
-        docker run --rm -d --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) \
-            -u $(id -u):$(id -g) -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
-            --model_path /opt/model --model_name resnet --port 9001 \
-            --target_device AUTO
-```
+@sphinxdirective
+.. code-block:: sh
+
+    docker run --rm -d --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) \
+    -u $(id -u):$(id -g) -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
```
Comment (mzegla) [inline docs/accelerators.md:198]: We should probably use GPU image here

File: docs/accelerators.md [hunk 3]
```diff
@@ -202,23 +209,29 @@ To enable Performance Hints for your application, use the following command:
...
-            -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
-            --model_path /opt/model --model_name resnet --port 9001 \
-            --plugin_config '{"PERFORMANCE_HINT": "LATENCY"}' \
-            --target_device AUTO
-```
+@sphinxdirective
+.. code-block:: sh
+
+    docker run --rm -d --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \
+    -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
```
Comment (mzegla) [inline docs/accelerators.md:216]: We should probably use GPU image here

File: docs/accelerators.md [hunk 4]
```diff
@@ -202,23 +209,29 @@ To enable Performance Hints for your application, use the following command:
...
-            -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
-            --model_path /opt/model --model_name resnet --port 9001 \
-            --plugin_config '{"PERFORMANCE_HINT": "THROUGHPUT"}' \
-            --target_device AUTO
-```
+@sphinxdirective
+.. code-block:: sh
+
+    docker run --rm -d --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \
+    -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001 openvino/model_server:latest \
```
Comment (mzegla) [inline docs/accelerators.md:229]: We should probably use GPU image here

**Review by mzegla — APPROVED** [review]

File: docs/accelerators.md [hunk 5]
```diff
@@ -23,10 +23,13 @@ To start the server with Neural Compute Stick use either of the two options:
...
-   ```
+@sphinxdirective
+.. code-block:: sh
+
+    docker run --rm -it -u 0 --device-cgroup-rule='c 189:* rmw' -v ${PWD}/models/public/resnet-50-tf:/opt/model -v /dev/bus/usb:/dev/bus/usb -p 9001:9001 openvino/model_server \
+    --model_path /opt/model --model_name resnet --port 9001 --target_device MYRIAD
+
+@endsphinxdirective
 
 2. less securely, in the docker privileged mode and mounting all devices.
```
Comment (dkalinowski) [inline docs/accelerators.md:34]:
```suggestion
2. Less securely, in the docker privileged mode and mounting all devices.
```

File: docs/model_server_c_api.md [hunk 6]
```diff
@@ -39,11 +39,10 @@ To process response, first you must check for inference error. If no error occur
 
 ## Preview limitations
 * Launching server in single model mode is not supported. You must use configuration file.
-* DAG pipelines cannot be used directly through C API. DAG inferences can be scheduled only by gRPC/HTTP endpoints.
-* There is no support for native file format (jpg/png) through C API.
+* There is no direct support for jpeg/png encoded input format through C API.
 * There are no server live, server ready, model ready, model metadata, metrics endpoints exposed through C API.
-* Inference scheduled through C API does not have inference success/failure, request time metrics counted.
```
Comment (dkalinowski) [inline docs/model_server_c_api.md:45]: C-API counts some latency related metrics. It does not count `request_time` only.

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**